Rubrik/Public/Get-RubrikDatabaseMount.ps1

```diff
@@ -0,0 +1,93 @@
+﻿#Requires -Version 3
+function Get-RubrikDatabaseMount
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and retrieves details on mounts for a SQL Server Database
+            
+      .DESCRIPTION
+      The Get-RubrikMount cmdlet will accept one of several different query parameters
+      and retireve the database Live Mount information for that criteria.
+
+      .NOTES
+      Written by Mike FAl for community usage
+      Twitter: @Mike_Fal
+      GitHub: MikeFal
+            
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+            
+      .EXAMPLE
+      Get-RubrikDatabaseMount
+      This will return details on all mounted databases.
+
+      .EXAMPLE
+      Get-RubrikDatabaseMount -id '11111111-2222-3333-4444-555555555555'
+      This will return details on mount id "11111111-2222-3333-4444-555555555555".
+
+      .EXAMPLE
+      Get-RubrikDatabaseMount -source_database_id (Get-RubrikDatabase -HostName FOO -Instance MSSQLSERVER -Database BAR).id
+      This will return details for any mounts found using the id value from a database named BAR on the FOO default instance.
+                  
+      .EXAMPLE
+      Get-RubrikDatabaseMount -source_database_name BAR
+      This returns any mounts where the source database is named BAR.
+
+      .EXAMPLE
+      Get-RubrikDatabaseMount -mounted_database_name BAR_LM
+      This returns any mounts with the name BAR_LM
+  #>
+
+  [CmdletBinding()]
+  Param(
+    # Rubrik's id of the mount
+    [Parameter(ValueFromPipelineByPropertyName = $true)]
+    [String]$id,
+    # Filters live mounts by database source id
+    [String]$source_database_id,
```

Comment:
For user-facing params, please use PascalCase with an alias that uses the snake_case needed by the API